doc/page-features.h

```diff
@@ -151,4 +151,22 @@ foo.userData<Pod>().i;  // 42
 An interface implementation can expose a structure containing some user data. This user data is
 preserved when the structure is copied. Thus it can be made available in a different interface.
 
+\section ServerImplementation   Implementation of Server inside C++ Implementation of Interface 
```

Comment:
Remove trailing whitespace.
Aim for shorter headline.

Reply:
"Implementation of Server inside C++ Implementation of Interface" makes no sense to me.

You are describing how to use an IPC proxy from any kind of C++ code.

Reply:
@jacky309  here idea is to tell the end user, how to create a c++ server without enabling IPC.enable:true , how would one put such task into words?  can you suggest?